app/controllers/businesses_controller.rb

```diff
@@ -1,6 +1,10 @@
 class BusinessesController < ApplicationController
   def show
-    @business = Business.first
+    @business = Business.find(params[:id])
     @items = ["250x300", "300x300", "200x200", "200x300"]
   end
+
+  def new  #corresponding create action is in Uesr::BusinessController
```

Comment:
I don't think it's unhappy about having a comment. It's unhappy because there shouldn't be a space between the `end` and `def`, and there should be a space after the `#` in the comment.